.travis.yml

```diff
@@ -30,4 +30,4 @@ notifications:
     channels:
       - "chat.freenode.net#mlpack"
     on_success: change
-    on_failure: always
+on_failure: always
```

Comment:
Hm, this should align with the parameter above, not sure it will even work that way.

Reply:
Hmm. I really need to configure my editor :/